.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.0.5
+current_version = 0.1.0
```

Comment:
I guess I need to change it here as well then? And in any place the bump tool will look for?

Reply:
Yes.